tests/Functional/Admin/PageAdminTest.php

```diff
@@ -262,12 +262,14 @@ private function prepareData(): void
 
         $parentBlock = new SonataPageBlock();
         $parentBlock->setType('sonata.page.block.container');
-        $parentBlock->setPage($page);
+        $parentBlock->setSetting('code', 'content');
```

Comment:
with this we can see what happens when a block is not orphan and is related to a section of the page.